libs/feature/editor/src/lib/components/wizard-field/wizard-field.component.ts

```diff
@@ -103,15 +104,20 @@ export class WizardFieldComponent implements AfterViewInit, OnDestroy {
         return data ? new Date(Number(data)) : new Date()
       }
       case WizardFieldType.DROPDOWN: {
+        //TODO called continuously
```

Comment:
Could you maybe make that clearer? does that mean that this code is executed a lot and that it's undesirable?